contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -3609,3 +3609,8 @@ IN query text)
 RETURNS SETOF RECORD
 AS 'babelfishpg_tsql', 'openquery_internal'
 LANGUAGE C VOLATILE;
+
+CREATE OR REPLACE FUNCTION sys.EOMONTH(date,int DEFAULT 0)
+RETURNS date
+AS 'babelfishpg_tsql', 'EOMONTH'
+LANGUAGE C IMMUTABLE STRICT;
```

Comment:
I think this function is also parallel safe and stable .

Reply:
Yes, will make the change in the next revision.

Reply:
Will make the change

Reply:
Have changed it to `LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
`